AutomatedTesting/Gem/PythonCoverage/Code/Source/PythonCoverageEditorSystemComponent.cpp

```diff
@@ -118,20 +117,21 @@ namespace PythonCoverage
     {
         AZStd::string contents;
 
-        // Compile the coverage for all test cases in this script
-        for (const auto& [testCase, entityComponents] : m_entityComponentMap)
+        // Compile the coverage for this test case
+        const auto coveringModules = GetParentComponentModulesForAllActivatedEntities(m_entityComponents);
+        if (coveringModules.empty())
         {
-            const auto coveringModules = GetParentComponentModulesForAllActivatedEntities(entityComponents);
-            if (coveringModules.empty())
-            {
-                return;
-            }
+            return;
+        }
 
-            contents = testCase + "\n";
-            for (const auto& coveringModule : coveringModules)
-            {
-                contents += AZStd::string::format(" %s\n", coveringModule.c_str());
-            }
+        contents += m_parentScriptPath + "\n";
+        contents += m_scriptPath + "\n";
+        contents += m_testFixture + "\n";
+        contents += m_testCase + "\n";
```

Comment:
suggestion: `AZStd::string::format` is till likely preferable here due to the number of temporaries this creates

Reply:
fixed 👍